### ISSUE TITLE
After opening the Dialog NVDA is reading twice

### ISSUE THREAD

**Issue by Kumaraswamy9096** [OfficeDev/office-ui-fabric-react#8436]
### Environment Information 
 
- __Package version(s)__: (6.117.0) 
- __Browser and OS versions__: (72.0.3626.121 and 1703)

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/Kumaraswamy9096/pen/GePjKX
 

 
#### Actual behavior:
 After opening the Dialog NVDA is reading Title and close button two times
 
#### Expected behavior:
 After opening the Dialog NVDA has to read the title and close button only once.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)

**Comment by khmakoto** [comment]
Hi, this seems to be a duplicate of #8350. I'll be using that issue to track this and close this as a duplicate.
